html/.gitignore

```diff
@@ -8,6 +8,5 @@ php-task5/images/
 php-task6/images/
 php-task6/mailer.php
 php-advance-2/mailer.php
-db/connection.php
```

Comment:
Why this file was removed?

Reply:
As this file containing the password of my MYSQL server so that I add this into `.gitignore` file to maintain privacy .